microsoft-365/enterprise/ms-cloud-germany-transition-add-experience.md

```diff
@@ -20,40 +20,28 @@ ms.custom:
 description: "Summary: Additional customer experience information when moving from Microsoft Cloud Germany (Microsoft Cloud Deutschland) to Office 365 services in the new German datacenter region."
 ---
 
-# Migration phases actions and impacts for the migration from Microsoft Cloud Deutschland (advanced) 
+# Migration phases actions and impacts for the migration from Microsoft Cloud Deutschland (advanced)
 
 Tenant migrations from Microsoft Cloud Deutschland to the Germany region of Microsoft's Office 365 services are executed as a set of phases and their configured actions for each workload. This figure shows the nine phases of migration to the new German datacenters.
 
 ![The nine phases of migration to the new Germany datacenters](../media/ms-cloud-germany-migration-opt-in/migration-organization.png)
 
 The following sections provide additional information on customer experiences when moving from Microsoft Cloud Germany (Microsoft Cloud Deutschland) to Office 365 services in the new German datacenter region.
 
-## Office 365 Portal Services
+## Office 365 Portal Services between phase 2 and phase 3
 
-Between Phase 2 of 9 and Phase 3 of 9, Partner Portal may not be accessible. During this time, Partner may not be able to access the tenants information on the Partner Portal. Since each migration is different, the duration of in-accessibility could be in hours.
+Between Phase 2 and phase 3, Partner Portal may not be accessible. During this time, Partner may not be able to access the tenants information on the Partner Portal. Since each migration is different, the duration of in-accessibility could be in hours.
```

Comment:
```suggestion
Between Phase 2 and phase 3, Partner Portal may not be accessible. During this time, Partner may not be able to access the tenant's information on the Partner Portal. Since each migration is different, the duration of in-accessibility could be in hours.
```